core/extrinsics_submission_service/service_configuration.hpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_SERVICE_CONFIGURATION_HPP
+#define KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_SERVICE_CONFIGURATION_HPP
+
+#include <cstdint>
+
+namespace kagome::service {
+  struct ExtrinsicSubmissionServiceConfiguration {
```

Comment:
make it nested struct inside ExtrinsicsSubmissionService, and call it Config

Reply:
sounds good

Reply:
done